src/shared/cpu-set-util.c

```diff
@@ -82,6 +82,23 @@ char *cpu_set_to_range_string(const CPUSet *set) {
         return TAKE_PTR(str) ?: strdup("");
 }
 
+char* cpu_set_to_mask_string(const CPUSet *a) {
+        _cleanup_free_ char *str = NULL;
+        unsigned mask = 0;
+        int i, r;
+
```

Comment:
there should be a comment here explaining which format precisely is used here, because it's a different one that we usually generate